ivre/db/__init__.py

```diff
@@ -1498,6 +1498,51 @@ def searchtorcert(self):
     def searchcertsubject(expr):
         raise NotImplementedError
 
+    @staticmethod
+    def searchdns(name, reverse=False, subdomains=False):
+        """Filters DNS records for domain `name`.
+        `name` can be a string, a list or a regular expression.
+        If `reverse` is set to True, filters reverse records.
+        If `subdomains` is set to True, the filter will match any subdomains.
+        """
+        raise NotImplementedError
+
+    def get(self, spec, **kargs):
+        """Queries the active column with the provided filter "spec",
+and returns a MongoDB cursor.
+
+This should be very fast, as no operation is done (the cursor is only
+returned). Next operations (e.g., .count(), enumeration, etc.) might
+take a long time, depending on both the operations and the filter.
+
+Any keyword argument is passed to the .find() method of the Mongodb
+column object, without any validation (and might have no effect if
+it is not expected)."""
+        raise NotImplementedError
+
+    def update_dns_blacklist(self):
+        """ Update the current database to allow to detect blacklist domains.
+This function insert a new element in the database, corresponding to the
```

Comment:
Can you update this comment to match (apart from the SQLite-specific part) the one from the SQLite backend?